generate-rss-feed.js

```diff
@@ -4,11 +4,18 @@ const __dirname = import.meta.dirname
 import { rssFeeder } from '@p-n-c/rssfeed'
 
 const src = path.join(__dirname, 'src')
+const root = 'src'
+const feed = {
+  title: 'People and Code, At Your Disposal',
+  link: 'https://people-and-code.com/',
+  description: 'Latest articles from People and Code',
+  outputPath: 'feed.rss',
+}
 const options = {
-  root: 'src',
   images: {
-    main: 'https://raw.githubusercontent.com/p-n-c/marketing/9b47dde787b5df57dd956277ca7154426ca2e80f/logo/PaC-logo.svg?token=AACAT4XXU6YMYIQWYMQK6CTHY5CNK',
+    main: 'https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Beardsley-peacockskirt.PNG/430px-Beardsley-peacockskirt.PNG',
```

Comment:
I like it but it's really massive on the RSS feed preview. Any chance you can constrain the size?

![image](https://github.com/user-attachments/assets/21bd0d20-2be4-4bc0-91a3-dc9f9f2f9ea0)

Reply:
The RSS recommended default is 144x400px (https://www.rssboard.org/rss-specification). 

But feed providers will crop images and the ones I've seen tend to be wider than they are tall though an older standard seems to be square…

We could use a large image such as 1200x600px or, to be more sustainable, 480x240px

Reply:
Yes small and sustainable is good for me!